datascience-notebook/onbuild/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM  jupyter/datascience-notebook
```

Comment:
Can you add the Jupyter copyright at the top of this file? You can copy/paste it from the other Dockerfiles.